buddyboss/buddyboss-platform
# PROD-923 - Backslash (\) in the forum post is not showing in Activity Feed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1337 .

### How to test the changes in this Pull Request:

1. Go to the Forum
2. Reply on an existing discussion or create a new discussion with backslash example: wp-content\themes\buddyboss-theme
3. Navigate to the Activity
4. And see the backslash is not showing

### Proof Screenshots or Video
https://www.loom.com/share/2466b711c1ed4630bf99b610c38d0e2c

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
